app/scss/questionAdmin.scss

```diff
@@ -0,0 +1,87 @@
+@import "imports/variables";
+
+.container {
+
+  .container_input {
+    float: left;
+    margin: 20px;
+
+    .btn {
+      margin: 5px;
+    }
+
+    .display-flex {
+      display: flex;
+      align-items: center;
+
+      input {
+        margin-right: 1rem;
+
+        &.primary-button {
+          font-size: 1.4rem;
+          color: $main-background-color;
+        }
+      }
+    }
+  }
+
+  .table {
+    font-size: 1.5rem;
+
+    .edit-question-button, .delete-question-button {
+      color: $main-background-color;
+    }
+
+    .edit-question-button {
+      background-color: $edit-button;
+    }
+
+    .delete-question-button {
+      background-color: $primary-button-hover;
+    }
+
+    &.question-table {
+      margin-top: 4rem;
+
+      tbody {
+        tr {
+          td {
+            vertical-align: middle;
+          }
+        }
+      }
+    }
+  }
+}
+
+.new-question-container {
+  margin-top: 3rem;
+  display: flex;
+  height: 150px;
+  flex-direction: column;
+  justify-content: space-between;
+  margin-bottom: 30px;
+}
+
+.overlay {
+  background: rgba(0, 0, 0, 0.6);
+  position: fixed;
+  z-index: 500;
```

Comment:
doesnt need to be this high - go for a scale of even numbers between 0 and 20 for z-index

Reply:
done